client/rpc/src/author/mod.rs

```diff
@@ -215,7 +215,7 @@ impl<P, Client> AuthorApi<TxHash<P>, BlockHash<P>> for Author<P, Client>
 				Ok(watcher) => {
 					subscriptions.add(subscriber, move |sink| {
 						sink
-							.sink_map_err(|_| unimplemented!())
+							.sink_map_err(|e| log::debug!("Subscription sink failed: {:?}", e))
```

Comment:
add a target maybe?

Reply:
The other logs in the crate also don't use any `target` :see_no_evil: 

But I also don't have a better target than the automatic `sc_rpc`. I could just come up with `sc-rpc` :see_no_evil:

Reply:
`sc-rpc` sounds good

Reply:
Let's do this in another PR if we really want it. None of the RPC logging calls in `client/rpc` have any target set, so if we want to change this we should update all of them.